web/profiles/custom/os2loop/themes/os2loop_theme/templates/field/user-name.html.twig

```diff
@@ -26,10 +26,12 @@
 #}
 <div class="col-md">
   <div>
-    <a href="{{ url('entity.user.canonical', {'user': uid}) }}">{{ account.os2loop_user_given_name.value }} {{ account.os2loop_user_family_name.value }}</a>
+    <a href="{{ url('entity.user.canonical', {'user': uid}) }}">{{ drupal_field('os2loop_user_given_name', 'user', uid)['#items'].0.value }} {{ drupal_field('os2loop_user_family_name', 'user', uid)['#items'].0.value }}</a>
```

Comment:
Sometimes I hate Drupal! Do we really need `['#items'].0.value`?
Can https://www.drupal.org/project/twig_field_value help clean this up?

Reply:
Looks like it could yes.

Reply:
After a rewrite with user-info.html.twig, it was made possible to grab the values from the user entity instead of calling drupal_field(), that turns:
`drupal_field('os2loop_user_given_name', 'user', uid)['#items'].0.value`
into
`user['#user'].os2loop_user_given_name.value`